src/Marketplace/MarketplaceScreen.js

```diff
@@ -35,6 +55,13 @@ export default function MarketplaceScreen() {
       >
         BUTTON
       </Button>
+      <Card style={styles.card}>
+        <Card.Title title={card.fields.Title} subtitle={card.fields.Subtitle} />
+        <Card.Cover source={{ uri: 'https://radio-images.npo.nl/%7Bformat%7D/45ed5691-8bc1-4018-9d67-242150cff944/954ab7c1-23a6-4d11-b30e-dc3abc3b37ec.jpg' }} />
```

Comment:
Don't use url images, if you want to use a dummy image download it and put it inside assets/images (might want to check with Grace since she's already done this and make sure the folder is named the same way to avoid merge issues)